docs/pages/dms/client.mdx

```diff
@@ -612,104 +673,111 @@ Set the `env` client option to `dev` while developing. Set it to `production` be
       <td>Add codecs to support additional content types.</td>
-      <td></td>
     </tr>
     <tr>
       <td>maxContentSize</td>
-      <td>100M</td>
+      <td>`100M`</td>
       <td>Maximum message content size in bytes.</td>
-      <td></td>
     </tr>
     <tr>
       <td>preCreateIdentityCallback</td>
-      <td>undefined</td>
+      <td>`undefined`</td>
       <td>
         preCreateIdentityCallback is a function that will be called immediately
         before a [Create Identity wallet
         signature](/protocol/signatures#sign-to-create-an-xmtp-identity) is
         requested from the user.
       </td>
-      <td></td>
     </tr>
     <tr>
       <td>preEnableIdentityCallback</td>
-      <td>undefined</td>
+      <td>`undefined`</td>
       <td>
         preEnableIdentityCallback is a function that will be called immediately
         before an [Enable Identity wallet
         signature](/protocol/signatures#sign-to-enable-an-xmtp-identity) is
         requested from the user.
       </td>
-      <td></td>
     </tr>
   </table>
 </div>
 
 <div data-title="Kotlin">
 <table>
-<tr><th>Parameter</th><th>Default</th><th>Description</th><th></th></tr>
-<tr><td>appVersion</td><td>`undefined`</td><td>Add a client app version identifier that's included with API requests. **Production apps are strongly encouraged to set this value.**<br/>For example, you can use the following format: `appVersion: APP_NAME + '/' + APP_VERSION`.<br/>Setting this value provides telemetry that shows which apps are using the XMTP client SDK. This information can help XMTP developers provide app support, especially around communicating important SDK updates, including deprecations and required upgrades.</td><td></td></tr>
-<tr><td>env</td><td>`DEV`</td><td>Connect to the specified XMTP network environment. Valid values include `DEV`, `.PRODUCTION`, or `LOCAL`. For important details about working with these environments, see [XMTP `production` and `dev` network environments](#xmtp-network-environments).</td><td></td></tr>
+  <tr>
+    <th>Parameter</th>
+    <th>Default</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>env</td>
+    <td>`DEV`</td>
+    <td>Connect to the specified XMTP network environment. Valid values include `DEV`, `.PRODUCTION`, or `LOCAL`. For important details about working with these environments, see [XMTP production and dev network environments](#xmtp-network-environments).</td>
+  </tr>
+  <tr>
+    <td>enableV3</td>
+    <td>`false`</td>
+    <td>To support group chat (a feature of XMTP v3), set to `true`. As a best practice, regardless of whether your app provides group chat, set `enableV3` to `true` to benefit from other aspects of the upgraded XMTP v3 protocol.</td>
+  </tr>
+  <tr>
+    <td>appContext</td>
+    <td>null</td>
+    <td>Required if `enableV3` is set to `true`.</td>
+  </tr>
+  <tr>
+    <td>dbEncryptionKey</td>
+    <td>null</td>
+    <td>Optional. If not provided, a key is created and stored for you.</td>
```

Comment:
This is now Required. We don't create or store keys anymore.

Reply:
We could link to the recommended key storage in kotlin https://developer.android.com/privacy-and-security/keystore